### PR TITLE
Declare dependence in parent pom

### PR DESCRIPTION
**Pull request by zhongjiajie**
---

## Purpose of this pull request

This patch unify declare dependence in parent pom.xml

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason: no need
* [x] Change does not need document change

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -327,6 +494,26 @@
...
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <attach>false</attach>
+                        <descriptors>
+                            <descriptor>assembly.xml</descriptor>
+                        </descriptors>
+                    </configuration>
+                </plugin>
```
Comment (zhongjiajie) [inline pom.xml:515]: I am not quit sure whether we should pop plugin version to parent `pom.xml`. It only belong to `seatunnel-dist` @CalvinKirs

**Review by CalvinKirs — APPROVED** [review]
LGTM

File: pom.xml (hunk 1 above)
Comment (CalvinKirs) [reply pom.xml:515]: I need to re-modify the packaging method later, and shade is not a good way

**Comment by zhongjiajie** [comment]
Thanks~

**Pull request merged**